cmd/collector/app/builder/builder_flags.go

```diff
@@ -38,4 +38,6 @@ var (
 	CollectorPort = flag.Int("collector.port", 14267, "The tchannel port for the collector service")
 	// CollectorHTTPPort is the port that the collector service listens in on for http requests
 	CollectorHTTPPort = flag.Int("collector.http-port", 14268, "The http port for the collector service")
+	// HealthCheckHTTPPort is the port that the health check service listens in on for http requests
+	HealthCheckHTTPPort = flag.Int("health-check.http-port", 15268, "The http port for the health check service")
```

Comment:
Like most of the code on this PR, this is duplicated, with one version for the collector and one for the query. This is because we might want to evolve the health checks for each module to be unique, as the query might have different concerns as the collector: the collector might still ingest data and put it on a temp buffer, while the query might not even be able to retrieve data to the UI.